### PR TITLE
chore: install `source-map-support`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
See https://github.com/remix-run/remix/pull/7009 & https://github.com/remix-run/remix/pull/7010

### CODE REVIEW

**Comment by kentcdodds** [comment]
Thank you!

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "kentcdodds.com",
   "version": "1.0.0",
   "engines": {
-    "node": "18",
+    "node": "16",
```
Comment (kentcdodds) [inline package.json:5]: I just noticed this change. Is this intentional?

Comment (MichaelDeBoey) [reply package.json:5]: Oh no that wasn't intentional, sorry

You can revert that if you want

Comment (kentcdodds) [reply package.json:5]: No worries :) Thanks!

**Pull request merged**